src/components/Button/Button.tsx

```diff
@@ -0,0 +1,16 @@
+import * as S from "./Button.style";
+
+type ButtonProps = {
+  children: React.ReactNode;
+};
+
+function Button(props: ButtonProps) {
```

Comment:
여기서도 구조분해할당으로 이렇게 가져와줄수있답니다! ㅎㅎ

```suggestion
function Button({ children }: ButtonProps) {
```